Content.Server/_Goobstation/Magic/ImmovableVoidRodComponent.cs

```diff
@@ -7,4 +7,6 @@ public sealed partial class ImmovableVoidRodComponent : Component
 {
     [DataField] public TimeSpan Lifetime = TimeSpan.FromSeconds(1f);
     public float Accumulator = 0f;
+    [DataField] public string SnowWallPrototype = "WallIce";
+    [DataField] public string IceTilePrototype = "FloorAstroIce";
```

Comment:
feels a little bad to re-define the same things that were defined in `AristocratComponent`